### DISCUSSION TITLE
[Feature]: When will version 6.0 be released?

### DISCUSSION THREAD

**Discussion by zc9** [fabricjs/fabric.js#9196] [Q&A]
### CheckList

- [X] I agree to follow this project's Code of Conduct
- [X] I have read and followed the [Contributing Guide](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md)
- [X] I have searched and referenced existing issues, feature requests and discussions
- [X] I am filing a **FEATURE** request.

### Description

When will version 6.0 be released?

### Current State

When will version 6.0 be released?

**Comment by zhe-he** [comment]
rc都还没发布呢，急什么

**Comment by asturur** [comment]
We have a last api change to review and merge and at least 2 bugs to fix.
We also have to update the website.

The released version won't change dramatically from current master

**Comment by sevgit** [comment]
@asturur I have a project currently on `v4.2.0` and was meaning to upgrade to `v5.3.0` this week. Would you recommend holding on and jumping straight to `v6.0.0` once its released?
